content/09.evolution.md

```diff
@@ -69,31 +69,98 @@ Research in response to prior outbreaks of HCoV-borne infections, such as SARS a
+They stated that they found an association between homozygosity for the SNP of interest and the severity of COVID-19.<!--Note: in-line stats differ from those in the associated table-->
 A follow-up study demonstrated worldwide variation in the frequency of these SNPs [@doi:10.1007/s00251-020-01188-0], and subsequent studies claimed to support this result by comparing the frequency of the SNP in different groups to the COVID-19 case fatality rate in those groups; they examined SNPs in several candidate genes and identified an association with another SNP in _IFITM3_ [@doi:10.3390/genes12010042].
 However, in the original study, the population-level frequency of the risk allele was consistent with its frequency in the mild population [@doi:10.1038/ncomms2433].
 A similar analysis examined both SNPs in Britons of different ancestral backgrounds and also reported a correlation [@doi:10.7717/peerj.10402].
 While this gene has been investigated for functions potentially relevant to COVID-19 pathogenesis by other groups as well (e.g., [@doi:10.15252/embj.2020106501; @doi:10.1080/22221751.2020.1787797], a follow-up analysis in Germany evaluted the effect of in 239 cases and 252 controls and reported non-significant effects [@doi:10.7717/peerj.10402].
 The narrative surrounding _IFITM3_ therefore reflects a broad methodological critique about candidate gene studies, where results often fail to replicate [@doi:10.1038/nrg796].
+The region associated with this gene was not identified in the large-scale GWAS conducted by the COVID-19 Host Genetics Initiative (COVID-19 HGI) [@doi:10.1038/s41586-021-03767-x], which is described in more detail below.
+
+A second source of genetic variability that was hypothesized to have an effect on COVID-19 outcomes were human leukocyte anitgens (HLA), or the major histocompatibility complex (MHC).
+Both MHC classes I and II play a critical role in both the innate and adaptive immune system because they are a pivotal component of antigen presentation.
+HLA classes I and II are also the most polymorphic loci in the human genome [@doi:10.1111/j.1399-0039.2012.01881.x].
+Additionally, because HLA polymorphisms are associated with geographic ancestry, study location and participant background offers important context [@doi:10.1371/journal.pone.0097282].
+Given the important role of the HLA complex in the immune response and the standing variation in the human population, HLA variation has been investigated for potential associations with COVID-19 outcomes.
+
+Several approaches have been taken to evaluate a potential role of HLA in COVID-19.
+_In silico_ analysis suggested one particular HLA locus that could affect binding of SARS-CoV-2 peptides to MHC class II [@doi:10.1016/j.ijid.2020.07.016].<!-- HLA-DRB1*04:01,         HLA-DPA1*01:03/DPB1*02:01-->
```

Comment:
What's with the greyed out bits?

Reply:
@RLordan I wrote the specific alleles in the comments in case they were useful in synthesizing the studies, I can remove them now!
```suggestion
_In silico_ analysis suggested one particular HLA locus that could affect binding of SARS-CoV-2 peptides to MHC class II [@doi:10.1016/j.ijid.2020.07.016].
```